extension/org.winepak.Platform.Extension.dxvk-61.yml

```diff
@@ -0,0 +1,95 @@
+build-extension: true
+
+id: org.winepak.Platform.Extension.dxvk-61
+branch: 3.0
+
+runtime: org.winepak.Platform
+runtime-version: 3.0
+sdk: org.winepak.Sdk
+
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: dxvk
+    buildsystem: simple
+    build-commands:
+      - install -d ${FLATPAK_DEST}/bin/
+      - install -d ${FLATPAK_DEST}/dlls/x32/
+      - install -d ${FLATPAK_DEST}/dlls/x64/
+      - install -D x32/d3d11.dll ${FLATPAK_DEST}/dlls/x32/
```

Comment:
Install the whole `x32` & `x64` so we also get the `setup_dxvk.sh` file (reason explained below).